components/bio-formats-plugins/src/loci/plugins/out/Exporter.java

```diff
@@ -560,6 +559,34 @@ else if (proc instanceof ShortProcessor) {
                 }
             }
 
+            //Check if the outputfiles
+            File dir = new File(outfile).getParentFile();
+            String[] files = dir.list();
+            boolean in = false;
+            for (int i = 0; i < outputFiles.length; i++) {
+                String name = new File(outputFiles[i]).getName();
+                for (int j = 0; j < files.length; j++) {
+                    if (files[j].equals(name)) {
+                        in = true;
+                        break;
+                    }
+                }
+            }
```

Comment:
Yes, I should have done that, I was testing the file output and did not simplify.
Too quick to move to other tasks